addons/sale_management/static/src/js/sale_management.js

```diff
@@ -19,28 +19,61 @@ publicWidget.registry.SaleUpdateLineButton = publicWidget.Widget.extend({
     _onClick: function (ev) {
         ev.preventDefault();
         var self = this;
-        var href = this.$el.attr("href");
-        var orderID = href.match(/my\/orders\/([0-9]+)/);
-        var lineID = href.match(/update_line\/([0-9]+)/);
+        var $input = $(ev.currentTarget);
+        this.$input = $input;
+        var orderID = $input.data('order-id');
+        var lineID = $input.data('line-id');
         var params = {
-            'line_id': lineID[1],
-            'remove': self.$el.is('[href*="remove"]'),
-            'unlink': self.$el.is('[href*="unlink"]'),
+            'line_id': parseInt(lineID),
+            'remove': $input.data('remove'),
+            'unlink': $input.data('unlink'),
         };
-        var token = href.match(/token=(.*)/);
-        if (token) {
-            params['access_token'] = token;
-        }
-        var url = "/my/orders/" + parseInt(orderID[1]) + "/update_line";
+        var url = "/my/orders/" + parseInt(orderID) + "/update_line";
         this._rpc({
             route: url,
             params: params,
         }).then(function (data) {
-            if (!data) {
-                window.location.reload();
+            if (!data['quantity']) {
+                self.$el.find('.sale_order_portal').empty();
+                self.$el.find('.sale_order_portal').append($(data['sale_order_portal_content']).children());
             }
-            self.$el.closest('.input-group').find('.js_quantity').val(data[0]);
-            $('[data-id="total_amount"] > span').html(data[1]);
+            else {
```

Comment:
Else on the same line as the closing brace of the if, see https://github.com/odoo/odoo/wiki/Javascript-coding-guidelines